transform/snowflake-dbt/models/blp/licenses.sql

```diff
@@ -24,8 +24,9 @@ WITH license_new AS (
       ON s.customer = c.id
     WHERE COALESCE(c.metadata:"id"::VARCHAR, c.metadata:"cws-customer") NOT IN (SELECT customerid FROM {{ source('licenses', 'licenses') }} GROUP BY 1)
     AND COALESCE(c.metadata:"id"::VARCHAR, c.metadata:"cws-customer") IS NOT NULL
-    AND COALESCE(s.metadata:"sku"::VARCHAR, SPLIT_PART(s.plan:"name"::VARCHAR, ' ', 2)) IN
-      ('E20', 'E10')
+    AND (COALESCE(s.metadata:"sku"::VARCHAR, SPLIT_PART(s.plan:"name"::VARCHAR, ' ', 2)) IN
+      ('E20', 'E10') OR 
+      (s.plan:"name"::VARCHAR IN ('Self-Hosted Enterprise','Self-Hosted Professional')))
```

Comment:
@sri-byte would we also include Cloud Professional & Cloud Enterprise plans here?

Reply:
Cloud Licenses are fetched directly from Stripe via another source. We do not need to include them here.